boefjes/boefjes/docker_boefjes_runner.py

```diff
@@ -53,16 +53,22 @@ def run(self) -> None:
 
             # save container log (stderr) to bytes
             self.bytes_api_client.login()
-            self.bytes_api_client.save_raw(task_id, container_logs, stderr_mime_types)
+            task = self.scheduler_client.get_task(task_id)
+
+            if task.status == TaskStatus.RUNNING:
+                self.boefje_meta.ended_at = datetime.now(timezone.utc)
+                self.bytes_api_client.save_boefje_meta(self.boefje_meta)  # The task didn't create a boefje_meta object
```

Comment:
I think it would be good to add that the status means the container was finished running without sending data back.